packages/lisk-transaction-pool/test/test_data.ts

```diff
@@ -0,0 +1,61 @@
+const transferTransactions = [{
```

Comment:
you can create `lisk-transaction-pool/fixtures` folder and put the `json` there.
something like transactions module